README.md

```diff
@@ -1 +1,62 @@
 # Hubble plugin for Adobe XD
+
+[![LICENSE](https://badgen.net/badge/license/MIT/blue)][license]
+![Last commit](https://badgen.net/github/last-commit/inthepocket/hubble-adobexd-plugin)
+
+Takes your design data out of Adobe XD and connects it to the Hubble atmosphere.
+
+# Contents
+
+- [Prerequisites](#prerequisites)
+- [Installing](#installing)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Development](#development)
+- [License](#license)
+
+## Prerequisites
+
+- Adobe XD (v21.0 or higher)
+- Hubble.app installed on your system
+
+## Installing
+
+TODO after plugin is released to the store
+
+## Usage
+
+After the installation, the 🔭 Hubble plugin will be available in your plugin list. The plugin has two main features.
+
+**Export to Hubble** will trigger an export in the Hubble app. Because Adobe does not allow to open 3rd party apps from within, the request has to be sent over Https and be redirected to a local Hubble request (hubble://). This is what a Trigger URL does and you will have to provide one after selecting a .xd file.
```

Comment:
```suggestion
The command __Export to Hubble__ will trigger an export to the Hubble app. Because Adobe does not allow to open 3rd party apps from within Adobe XD, the export request has to be sent via a [web proxy](https://github.com/inthepocket/hubble-web-proxy) to Hubble.app. Fill in the link to your web proxy (see hubble-web-proxy)[https://github.com/inthepocket/hubble-web-proxy] in the field __Trigger URL__
```